mlflow/pipelines/step.py

```diff
@@ -147,7 +147,7 @@ def inspect(self, output_directory: str):
 
         card = BaseCard.load(card_path)
         card_html_path = os.path.join(output_directory, CARD_HTML_NAME)
```

Comment:
We can remove this I think?

Reply:
Updated the PR. I was getting CI failures because a local run of the pipeline expects to use `html_file_path`...